templates/web/base/around/on_map_list_items.html

```diff
@@ -1,7 +1,14 @@
-[% IF on_map.size %]
-    [% FOREACH problem IN on_map %]
+[% all_reports = on_map.merge(around_map) %]
```

Comment:
It's not /that/ unuseful – it's the reports on the map first by last updated, followed by those off-map by distance, which presumably will confuse someone at some point but in general seems workable enough. I wondered about that, but I thought it might be all right (until someone minds) and clear enough to show that it is the template displaying these two lists together, rather than hold this down trying to come up with an appropriate sort ;)